radio/src/pulses/pxx_arm.cpp

```diff
@@ -348,10 +353,30 @@ void setupPulsesPXX(uint8_t port)
       }
     }
     else {
-      if (i < sendUpperChannels) {
-        int channel = 8 + g_model.moduleData[port].channelsStart + i;
-        int value = channelOutputs[channel] + 2*PPM_CH_CENTER(channel) - 2*PPM_CENTER;
-        pulseValue = limit(2049, (value * 512 / 682) + 3072, 4094);
+      if (g_model.moduleData[port].channelsCount > 8) {
+        int channel = g_model.moduleData[port].channelsStart + i;        
+        iError[port][channel] += ABS( (short)((channelOutputs[channel]) - lastPPM[port][channel]));
```

Comment:
but since this value is calculated in each loop, why do you need to store it in a static array?